src/app/actions/index.ts

```diff
@@ -1 +1,25 @@
-export { default as getTodayPrices } from './getTodayPrices';
+import getTodayPrices from './getTodayPrices';
+
+export class Actions {
```

Comment:
This is used as a global class to store the access token used for the actions. It's not clean and was thinking about using local storage but auth0 recommends to keep access tokens in memory.